levenshtein/php/code.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+function levenshtein_distance(string $str1, string $str2): int
+{
+    $m = strlen($str1);
+    $n = strlen($str2);
+  
+    // Create a matrix to store distances
+    $matrix = [];
```

Comment:
I didn't actually run this and I don't know php very well, but isn't this just a 1d array?

Reply:
Because it's dynamic language. It is allocated to 2d right in below loops. Theres no way to

Reply:
👍